generators/entity/index.js

```diff
@@ -120,13 +139,21 @@ module.exports = EntityGenerator.extend({
             this.filename = this.jhipsterConfigDirectory + '/' + _s.capitalize(this.name) + '.json';
             if (shelljs.test('-f', this.filename)) {
                 this.log(chalk.green('\nFound the ' + this.filename + ' configuration file, entity can be automatically generated!\n'));
-                try {
-                    this.fileData = this.fs.readJSON(this.filename);
-                } catch (err) {
-                    this.log(chalk.red('\nThe configuration file could not be read!\n'));
-                    return;
-                }
                 this.useConfigurationFile = true;
+                this.fromPath = this.filename;
+            }
+            else {
+                var filePath = this.jhipsterConfigDirectory + '/jhipster-remote-entities.json';
+                var allData = this.fs.readJSON(filePath);
```

Comment:
may be we can have it called `pathConfigs`and the enity key value as `pathConfig` or something similar